spec/controllers/books_controller_spec.rb

```diff
@@ -107,7 +107,16 @@
     let(:send_request){ put :update, params}
     it "updates the data in a Book object" do
       send_request
-      expect(assigns(:book).reload.title).to eq(valid_params[:title])
+      expect(assigns(:book).title).to eq(valid_params[:title])
```

Comment:
I thought I needed it, but I didn't. I just tried it without it to see if it would work, and it did.